### ISSUE TITLE
Test New Issue

### ISSUE THREAD

**Issue by nisan-jackjaffa** [Test-NF/Test001#3]
The "Issues" section is simple and clean here.
I'm assigning this issue to both of us.

**Comment by nisan-jackjaffa** [comment]
It seems there is only 2 Statuses:  Open and Closed.

**Comment by MintD** [comment]
Well, that sucks.

I love the VSTS Task board, and that it allows uploading to a repository directly from VS.

**Comment by nisan-jackjaffa** [comment]
I was able to "Clone" a GitHub Repo to my local VisualStudio.  Once that was done,  I copied some files in my local folder, and was then able to Commit/Push the changes to the GitHub Repository through VisualStudio.

**Comment by nisan-jackjaffa** [comment]
Yup, the board is nice on VS.  The files section is a lot better in VS.
It seems that  I can't assign any items to you on VS, probably a permission thing.
